src/electron/add_tap_device.bat

```diff
@@ -62,10 +63,16 @@ echo Found TAP device name: "%TAP_NAME%"
 call :wait_for_device "%TAP_NAME%"
 
 :: Attempt to rename the device even if waiting timed out.
-netsh interface set interface name= "%TAP_NAME%" newname= "%DEVICE_NAME%"
+netsh interface set interface name="%TAP_NAME%" newname="%DEVICE_NAME%"
 if %errorlevel% neq 0 (
-  echo Could not rename TAP device. >&2
-  exit /b %ERROR_TAP_RENAME%
+  :: Try to rename the device through powershell in case netsh failed due to not being able to "see"
```

Comment:
Would it make sense to log here?  That way if the fallback is correlated with some issue, we have a chance of figuring that out from user reports.  Or does the powershell invocation already generate distinctive log output?

Reply:
We already get the output (stdout & stderr) of the above command, so we'll know whether it failed. Additionally, powershell generates distinctive logging that would allow us to infer it was invoked.